jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java

```diff
@@ -48,7 +48,9 @@ public List<SecurityAlertsVH> fetchSecurityAlertsDetails(SubscriptionVH subscrip
 				securityAlertsVH.setType(databricksObject.get("type").getAsString());
 				securityAlertsVH.setSubscription(subscription.getSubscriptionId());
 				securityAlertsVH.setSubscriptionName(subscription.getSubscriptionName());
-				securityAlertsVH.setRegion(Util.getRegionValue(subscription,databricksObject.get("location").getAsString()));
+				String id=securityAlertsVH.getId();
+				String region=id.substring(id.indexOf("locations/")+new String("locations/").length(),id.indexOf("/alerts"));
+				securityAlertsVH.setRegion(Util.getRegionValue(subscription,region));
```

Comment:
Update according to code style in this file.

Reply:
you can use autoformat if you are using InteliJ

Reply:
Resolved in commit